### PR TITLE
Add ctrl+c copy in update report

### LINKED ISSUE #7834: Not possible to select data in update report in gui
Clicking the select button in the top left corner here: 
image

will select everything in the table, but it is not possible to copy everything at once.

### PR DESCRIPTION
**Pull request by frode-aarstad**
**Issue**
Resolves #7834 

**Approach**
image

- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Added appropriate release note label
- [ ] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [ ] Make sure tests pass locally (after every commit!)

## When applicable
- [ ] **When there are user facing changes**: Updated documentation
- [ ] **New behavior or changes to existing untested code**: Ensured that unit tests are added (See [Ground Rules](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md#ground-rules)).
- [ ] **Large PR**: Prepare changes in small commits for more convenient review
- [ ] **Bug fix**: Add regression test for the bug
- [ ] **Bug fix**: Create Backport PR to latest release

### CODE REVIEW

File: src/ert/gui/simulation/view/update.py [hunk 1]
```diff
@@ -1,12 +1,14 @@
 import time
 from collections import defaultdict
-from typing import DefaultDict
+from typing import DefaultDict, List
 
 import humanize
+from PyQt5.QtGui import QKeyEvent
```
Comment (dafeda) [inline src/ert/gui/simulation/view/update.py:6]:
```suggestion
from qtpy.QtGui import QKeyEvent
```

**Comment by dafeda** [comment]
What's the use-case for this? Perhaps @oyvindeide ?

**Comment by oyvindeide** [comment]
> What's the use-case for this? Perhaps @oyvindeide ?

It was already possible to mark everything, and copy, but only got the first cell, so did not do as I would expect. Also got asked during hm-course.

**Comment by dafeda** [comment]
> > What's the use-case for this? Perhaps @oyvindeide ?
> 
> It was already possible to mark everything, and copy, but only got the first cell, so did not do as I would expect. Also got asked during hm-course.

I can see why users would want to copy this information, but perhaps we should format it in a more usable fashion, like .csv or something?

**Comment by oyvindeide** [comment]
> > > What's the use-case for this? Perhaps @oyvindeide ?
> > 
> > 
> > It was already possible to mark everything, and copy, but only got the first cell, so did not do as I would expect. Also got asked during hm-course.
> 
> I can see why users would want to copy this information, but perhaps we should format it in a more usable fashion, like .csv or something?

Will output csv here: https://github.com/equinor/ert/pull/8041, though at my expectation for these kinds of tables is that it is possible to copy from them.

**Comment by dafeda** [comment]
> Will output csv here: #8041, though at my expectation for these kinds of tables is that it is possible to copy from them.

Alright 👍

**Comment by oyvindeide** [comment]
> Will output csv here: #8041, though at my expectation for these kinds of tables is that it is possible to copy from them.

Granted, it is a minor thing, but feel like it makes the UI more user friendly.

**Review by dafeda — APPROVED** [review]

**Pull request merged**